R/oauth-init.R

```diff
@@ -31,11 +31,11 @@ init_oauth1.0 <- function(endpoint, app, permission = NULL,
   authorize_url <- modify_url(endpoint$authorize, query = list(
     oauth_token = token,
     permission = "read"))
-  verifier <- oauth_listener(authorize_url, is_interactive)[[1]]
+  verifier <- oauth_listener(authorize_url, is_interactive)$oauth_verifier
```

Comment:
Ok I have updated it as suggested with https://github.com/hadley/httr/pull/373/commits/da2bef023f93f44e0c6166359fe88d0369cd7aad